deployments/helm-chart/values.yaml

```diff
@@ -20,7 +20,7 @@ controller:
     enable: false
 
   ## Enables the Ingress controller pods to use the host's network namespace.
-  hostNetwork: false
+  hostNetwork: true
```

Comment:
Hi Lorcan, why does this and the health check below need to be true now?

Reply:
these were a mistake, I forgot to remove them

Reply:
No problem :)